test/algorithms/relational_operations/relate/test_relate.hpp

```diff
@@ -67,6 +70,44 @@ std::string matrix_format(std::string const& matrix1, std::string const& matrix2
          + ( !matrix2.empty() ? " || " : "" ) + matrix2;
 }
 
+template <typename M>
+char get_ii(M const& m)
```

Comment:
It's for unit testing only. We don't use `inline` keyword in the test hpp files.